Library/Homebrew/utils.rb

```diff
@@ -124,20 +114,20 @@ def odie(error)
 def pretty_installed(f)
   if !$stdout.tty?
     "#{f}"
-  elsif ENV["HOMEBREW_NO_EMOJI"]
+  elsif !Emoji.enabled?
```

Comment:
I'd be typically aiming for a minimal diff, but since one of the bodies requires changes anyway, I figured the benefit of slightly more readable code would outweigh the minimally bigger diff. (I don't feel very strongly about this and I'd be completely happy with you leaving it as-is.)